django/contrib/admin/widgets.py

```diff
@@ -580,8 +580,8 @@ def media(self):
 
 
 class AutocompleteSelect(AutocompleteMixin, forms.Select):
-    pass
+    option_attrs = None
```

Comment:
The test `admin_views.tests.AdminViewBasicTest.test_trailing_slash_required` broke because `AutocompleteSelect` had no attribute `option_attrs`. 

But after some thinking, i think it's better to solve this in `ChoiceWidget.create_option() ` as i have done now:
```python
        if not hasattr(self, "option_attrs"):
            # AutocompleteSelect subclasses AutocompleteMixin first, and then Select
            # (which subclasses this class).
            # So the ChoiceWidget's __init__ method (which is how other Select classes
            # get their option_attrs set) is not called.
            # But AutocompleteSelect does call create_option, so we need to allow for
            # a missing option_attrs attribute.
            option_attrs = {}
```